repeat-loop.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+while true
+do
+  $(which kubectl) apply -f job.yaml
```

Comment:
I haven't tried it yet but I suspect that repeating `kubectl apply` won't actually re-run the job, because the job already exists with the same name.